packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx

```diff
@@ -76,7 +73,7 @@ const PatientTable: React.FC<PatientTableProps> = ({ patients, columns, search,
   if (isLoading) {
     return (
       <DataTableSkeleton
-        style={{ backgroundColor: 'transparent', padding: '0rem', margin: '1rem' }}
+        style={{ backgroundColor: 'transparent', padding: '0rem' }}
```

Comment:
I think this would be better in the SCSS file. But it's not critical, only if you have the time.